strawberryfields/__init__.py

```diff
@@ -75,5 +79,48 @@ def version():
     return __version__
 
 
-#: float: numerical value of hbar for the frontend (in the implicit units of position * momentum)
-hbar = 2
+def about():
+    """About box for Strawberry Fields.
+
+    Prints the installed version numbers for SF and its dependencies,
+    and some system info. Please include this information in bug reports.
+    """
+    import sys
```

Comment:
Any reason these are imported in the function and not at the top of the file?

Reply:
This will avoid these superfluous imports from occurring on standard Python import - they will only be imported when `about()` or `cite()` is called.

On repeated calls, they will not be re-imported.

I assume this is the reason, @smite?

Reply:
That's the idea, although I think most of them are imported anyway in the other modules.